uds/database/data_record/raw_data_record.py

```
@@ -1,123 +1,62 @@
-        """
-        Whether this Data Record might occur multiple times.
+    Raw Data Records are the most basic Data Records which does not contain translation between physical and raw values.
+    """
 
-        Values meaning:
-        - False - exactly one occurrence in every diagnostic message
-        - True - number of occurrences might vary
+    def __init__(self,
+                 name: str,
+                 length: int,
+                 children: Sequence[AbstractDataRecord] = tuple(),
```

Ruff (0.12.2)

Unnecessary `tuple()` call (rewrite as a literal)

Rewrite as a literal

(C408)